packages/yoga/src/Dialog/web/__snapshots__/Dialog.test.jsx.snap

```diff
@@ -121,7 +121,7 @@ exports[`<Dialog /> should match snapshot 1`] = `
   display: -webkit-flex;
   display: -ms-flexbox;
   display: flex;
-  z-index: 3;
+  z-index: 9;
```

Comment:
This snapshot should be reverted

Reply:
I already did the reversal, take a look at the modified files of this PR: https://github.com/gympass/yoga/pull/607/files